dali/pipeline/operators/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+add_subdirectory(arithmetic)
```

Comment:
Just a thought: I know we've grown to use the term "arithmetic", but it will eventually cover more. I wonder if we should rename it to "expr".

Reply:
I can do as a last step not to mix everything now.

Reply:
Done